src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/service/SubjectAccessRequestService.kt

```diff
@@ -1,6 +1,8 @@
 package uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.service
 
 import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClientResponseException.NotFound
```

Comment:
“reactive” looks suspicious here 👀

Reply:
As discussed offline it is what we currently handle in the exception handling file and it works.

I also tried to replace with `import org.springframework.web.client.HttpClientErrorException.NotFound` and it did not work, so it seems like the repository is throwing those exceptions from this package.